### PR TITLE
优化Masa Try 新增Masa Try Server 

### PR DESCRIPTION
**Pull request by 239573049**
# Description

_Please explain the changes you've made_

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: src/Masa.Try.Shared/Pages/Index.razor.cs [hunk 1]
```diff
@@ -61,7 +77,7 @@ public partial class Index : IDisposable
         automaticLayout = true,
     };
 
-    private DotNetObjectReference<Index> _objRef; 
+    private DotNetObjectReference<Index> DotNetObject;
```
Comment (capdiem) [inline src/Masa.Try.Shared/Pages/Index.razor.cs:80]: 怎么又改回DotNetObject？

File: src/Masa.Trt.Server/Program.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
+using Masa.Try.Shared;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddRazorPages();
+builder.Services.AddServerSideBlazor();
+
+builder.Services.AddMasaBlazor();
+builder.Services.AddScoped<TryJSModule>();
+
+builder.Services.AddScoped(sp => new HttpClient { BaseAddress = new Uri("http://localhost:5250") });
```
Comment (capdiem) [inline src/Masa.Trt.Server/Program.cs:10]: http://localhost:5250 这是？

**Review by capdiem — CHANGES_REQUESTED** [review]

File: src/Masa.Try.Shared/Pages/Index.razor [hunk 3]
```diff
@@ -1,45 +1,125 @@
 ﻿@page "/"
 @inject HttpClient HttpClient
+@inject TryJSModule TryJSModule
+<MApp Id="inspire">
```
[the remark this answers is not in the document]
Comment (239573049) [reply src/Masa.Try.Shared/Pages/Index.razor:4]: 我看你的模板这样写的

**Review by capdiem — APPROVED** [review]
approve temporarily

**Pull request merged**